Tests/Shared.Specs/AssertionScopeSpecs.cs

```diff
@@ -192,6 +192,22 @@ public void When_parentheses_are_used_in_the_because_arguments_it_should_render_
                 .WithMessage("*because can't use these in becauseArgs: { }*");
         }
 
+        [Fact]
+        public void When_invalid_format_is_used_in_because_parameter_it_should_render_default_text()
+        {
+            //-----------------------------------------------------------------------------------------------------------
+            // Act
+            //-----------------------------------------------------------------------------------------------------------
+            const string because = "use of {} is considered invalid in because parameter";
+            Action act = () => 1.Should().Be(2, because);
+
+            //-----------------------------------------------------------------------------------------------------------
+            // Assert
+            //-----------------------------------------------------------------------------------------------------------
+            act.Should().Throw<XunitException>()
+                .WithMessage($"*because message '{because}' could not be formatted with string.Format*");
```

Comment:
Tiny issue: you are reproducing the logic from production code in tests. I suggest avoiding $ formatting and just prepare expected message manually.

Reply:
I embedded the text directly in the expected message with the most recent commit.